docs/dashboard.asciidoc

```diff
@@ -3,163 +3,113 @@
-The request statistics show the query duration, request duration, total number
-of matching records, and the index (or index pattern) that was searched.
+. Open the dashboard.
+. Hover over the visualization and click the *Expand* button in the lower
+left.
++
+The underlying data is displayed in a table. You can also view
+the raw Elasticsearch request, the response in JSON, and the request statistics.
+These statistics show the query duration, request duration, total number
+of matching records, and the index that was searched.
 +
-image:images/NYCTA-Table.jpg[]
+[role="screenshot"]
+image:images/Dashboard_visualization_data.png[Example of visualization data]
 
-To export the data behind the visualization as a comma-separated-values
-(CSV) file, click the *Raw* or *Formatted* link at the bottom of the data 
-Table. *Raw* exports the data as it is stored in Elasticsearch. *Formatted*
-exports the results of any applicable Kibana <<managing-fields,field
+To export the data as a comma separated values
+(CSV) file, click *Raw* or *Formatted* at the bottom of the data
+table. *Raw* exports the response data as provided. *Formatted*
+exports the reponse data using applicable Kibana <<managing-fields,field
 formatters>>.
 
 To return to the visualization, click the *Collapse* button in the lower left
-corner of the container.
-
-[float]
-[[changing-the-visualization]]
-=== Modifying a Visualization
-
-To open a visualization in the Visualization Editor:
-
-. Enter Edit Mode.
-. Hover over it to display the container controls.
-. Click the *Edit* button in the upper right corner of the container.
-
-
-[[loading-a-saved-dashboard]]
-== Loading a Dashboard
-
-To open a saved dashboard:
-
-. Click *Dashboard* in the side navigation. 
-. Click *Open* and select a dashboard. If you have a large number of
-dashboards, you can enter a *Filter* string to filter the list. 
-+
-TIP: To import, export, and delete dashboards, click the *Manage Dashboards* link
-to open *Management/Kibana/Saved Objects/Dashboards*. 
+corner.
 
 [[sharing-dashboards]]
 == Sharing a Dashboard
 
-You can either share a direct link to a Kibana dashboard with another user,
+You can either share a direct link to a Kibana dashboard,
 or embed the dashboard in a web page. Users must have Kibana access
-to view embedded dashboards.
+to view an embedded dashboard.
 
 [[embedding-dashboards]]
-To share a dashboard:
 
-. Click *Dashboard* in the side navigation. 
 . Open the dashboard you want to share.
-. Click  *Share*.
-. Copy the link you want to share or the iframe you want to embed. You can 
-share the live dashboard or a static snapshot of the current point in time. 
+. In the menu bar, click *Share*.
+. Copy the link you want to share or the iframe you want to embed. You can
+share the live dashboard or a static snapshot of the current point in time.
 +
 TIP: When sharing a link to a dashboard snapshot, use the *Short URL*. Snapshot
```

Comment:
A common thing that trips people up is that users must have write access to .kibana to create short urls. May be worth calling out.

Another thing that might be worth calling out that isn't exposed in the UI at all but many people find handy is that you can create a link to a dashboard by title, by doing this:

`${domain}/${basepath?}/app/kibana#/dashboards?title=${yourdashboardtitle}`

It's not exposed in the UI because we can't figure out how to communicate it properly (maybe you will have some ideas!).  Because it's just linked by title and not id, changing titles on dashboards can change the link.  So you can't really say "link to this dashboard" because that might not be true later down the road. Here is the PR it was implemented : https://github.com/elastic/kibana/pull/14760